src/BuildIntegration/Microsoft.NETCore.Native.Unix.props

```diff
@@ -40,6 +40,7 @@ See the LICENSE file in the project root for more information.
   <ItemGroup>
     <AdditionalNativeLibrary Include="$(IlcPath)/sdk/libSystem.Private.CoreLib.Native.a" />
     <AdditionalNativeLibrary Include="$(IlcPath)/framework/System.Native.a" />
+    <AdditionalNativeLibrary Include="$(IlcPath)/framework/libSystem.Globalization.Native.a" />
```

Comment:
We have made this choice a long time ago so that the shared library name matches the corresponding managed assembly. You can see the quite long discussion here: https://github.com/dotnet/coreclr/pull/745